expire-first-party-tracking-cookies/tests.json

```diff
@@ -0,0 +1,117 @@
+            },
+            {
+                "name": "above treshold - expiry modified",
+                "siteURL": "https://example.com/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=example.com; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 86400,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "same origin - expiry not modified",
+                "siteURL": "https://tracker.com/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=tracker.com; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "same entity - expiry not modified",
+                "siteURL": "https://tracker.test/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=tracker.test; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": [
+                    "web-extension"
+                ]
+            },
+            {
+                "name": "site safelisted locally - expiry not modified",
+                "siteURL": "https://local-exception.test/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=local-exception.test; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "site safelisted globally - expiry not modified",
+                "siteURL": "https://global-exception.test/",
+                "scriptURL": "https://tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=global-exception.test; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "not a tracking script - expiry not modified",
+                "siteURL": "https://example.com/",
+                "scriptURL": "https://not-a-tracker.com/script.js",
+                "setDocumentCookie": "foo=bar; Max-Age=900000; Domain=example.com; Path=/; Secure",
+                "expectCookieSet": true,
+                "expectExpiryToBe": 900000,
+                "exceptPlatforms": []
+            },
+            {
+                "name": "above treshold - cnamed tracking script - expiry modified",
+                "siteURL": "https://example.com/",
+                "scriptURL": "https://tracker.cname.com/script.js",
```

Comment:
Could you also test scripts cnamed to same eTLD+1 as the site?

Reply:
good idea, added!